### PR TITLE
Dtsstci 103 - Complete Unit Tests in Special Tribs Frontend: Get and Post Controllers

### PR DESCRIPTION
**Pull request by Jefex7**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/DTSSTCI-103

### Change description ###

Complete Unit Tests in Special Tribs Frontend: Get and Post Controllers

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/app/controller/GetController.test.ts [hunk 1]
```diff
@@ -16,7 +13,8 @@ describe('GetController', () => {
       text: 'welsh',
     },
   };
-  //const userEmail = 'test@example.com';
+
+  // const userEmail = 'test@example.com'
```
Comment (gmmagruder) [inline src/main/app/controller/GetController.test.ts:17]: Can you remove this comment please?

File: src/main/steps/common/components/common/renderSubFields.test.ts [hunk 2]
```diff
@@ -22,17 +22,6 @@ describe('steps > common > components > common > renderSubFields', () => {
     //expect(subFields[radiosFieldName].type).toBe(radiosType);
```
Comment (rajaram-nagarajan) [inline src/main/steps/common/components/common/renderSubFields.test.ts:22]: This comment can be removed.

File: src/main/steps/common/components/common/renderSubFields.test.ts [hunk 3]
```diff
@@ -22,17 +22,6 @@ describe('steps > common > components > common > renderSubFields', () => {
...
   });
```
Comment (rajaram-nagarajan) [inline src/main/steps/common/components/common/renderSubFields.test.ts:24]: The constants radiosFieldName, radiosType and values can be removed. And their reference in renderSubFields can be removed as well.

File: src/main/steps/common/components/common/renderSubFields.test.ts [hunk 4]
```diff
@@ -22,17 +22,6 @@ describe('steps > common > components > common > renderSubFields', () => {
...
 
-  it('should correctly render all radios subfields', () => {
```
Comment (rajaram-nagarajan) [inline src/main/steps/common/components/common/renderSubFields.test.ts:25]: RadiosValues can be removed from the import.

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Review by tomxelliott — APPROVED** [review]

**Pull request merged**